CHANGELOG.md

```diff
@@ -27,25 +42,7 @@
   * Semantically changed `errors` to always have the details of one or many errors that have occurred. For example: "Unknown key received: :foobar while loading $.payload.user"
   * Note: if you are an application that used and tested against the previous `message` field you will need to adjust your tests to check for the values in the `summary` field and or the `errors` contents. But it will now be a much more consistent experience that will allow API clients to notify of the exact errors and details to their clients.
-* Added `Application.validation_handler` to customize response generation for validation errors. See [validation_handler.rb](lib/praxis/validation_handler.rb) for default version.
-* Copied mustermann's routers to praxis repo in anticipation of their removal from mustermann itself.
-* Added response body validation.
-  * Validation is controlled by the `praxis.validate_response_bodies` boolean
-config option, and uses the `media_type` defined for the response definition.
-* Added `location:` option to `Responses::Created.new`.
-* `ResourceDefinition.parse_href` now accepts any instance of `URI::Generic` in addition to a string.
-* Fixed path generation for nested ResourceDefinitions
-* Substantial changes and improvements to multipart request and response handling:
-  * Added `Praxis::Types::MultipartArray`, a type of `Attributor::Collection` with `Praxis::MultipartPart` members that allows the handling of duplicate named parts and unnamed ones. The new type supports defining many details about the expected structure of the parts, including headers, payloads, names and filename information. In addition, this type is able to load and validate full multipart bodies into the expected type structure, generate example objects as well as dump them into a full multipart body including boundaries and properly encoded parts following their content-types. See documentation for details and more features.
-  * Made `Praxis::MultipartPart` a proper `Attributor::Type`.
-  * Added `Praxis::Responses:MultipartOk` properly returning `MultipartArray` responses.
-  * Deprecated `Praxis::Multipart`. A replacement for true hash-like behavior will be added before their removal in 1.0.
-* `ActionDefinition#response` now accepts an optional second `type` parameter, and optional block to further define that type. The `type` provided will be used to specify the `media_type` option to pass to the corresponding `ResponseDefinition`.
-* The `header` directive inside `ActionDefinition#headers` now accepts an optional second `type` parameter that may be used to override the default `String` type that would be used.
-* Added `Praxis::Handlers::Plain` encoder for 'text/plain'.
-* Fixed `Praxis::Handlers::XML ` handler to transform dashes to underscores and treat empty hashes like ActiveSupport does.
-* Adds hierarchival navigation to the doc browser.
-* Adds a ConfigurationProvider allowing for easy doc customization.
```

Comment:
No it will not. There are still a few things being worked in the PR. This is probably some weird rebasing error. Bottomline, no, nothing in here should be removed.